docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -99,10 +99,11 @@ listen ceph_rgw
 7: br-ex	inet 10.0.0.106/24 brd 10.0.0.255 scope global br-ex\   	valid_lft forever preferred_lft forever
 ...
 ----
+.Note: When target nodes are not managed by Director, it's not possible to use this procedure/tool to configure the network. All the required networks must be manually configured by the administrator.
```

Comment:
I'm not sure `Notes` have this syntax: @klgill to check and ack/nack the proposed content.